potato/server_utils/config_module.py

```diff
@@ -3,13 +3,62 @@
 """
 
 import yaml
+import os
 
 config = {}
 
 
 def init_config(args):
     global config
-    with open(args.config_file, "r") as file_p:
+
+    project_dir = os.getcwd() #get the current working dir as the default project_dir
+    config_file = None
+    # if the .yaml config file is given, directly use it
+    if args.config_file[-5:] == '.yaml':
+        if os.path.exists(args.config_file):
+            config_file = args.config_file
+            print("Warning: when you run the server directly from a .yaml file, you must first go the the proper directory before running it")
```

Comment:
I don't understand this error message

Reply:
fixed with [bd9e469](https://github.com/davidjurgens/potato/pull/32/commits/bd9e46940e93ddaecb4079c9b08e81e904e6941c)